corehq/apps/export/views.py

```diff
@@ -1512,26 +1526,40 @@ def fmt_export_data(self, export):
         if isinstance(export, FormExportSchema):
             emailed_export = self.get_formatted_emailed_export(export)
             is_legacy = True
+            can_edit = True
+            description = ''
+            my_export = None
+            sharing = None
+            owner_username = 'unknown'
```

Comment:
nit: could be worth making this a constant, since it's being used in couple of places.